src/t8_geometry/t8_geometry_implementations/t8_geometry_linear_axis_aligned.cxx

```diff
@@ -82,6 +82,13 @@ t8_geometry_linear_axis_aligned::t8_geom_point_batch_inside_element (t8_forest_t
   return;
 }
 
+bool
+t8_geometry_linear_axis_aligned::t8_geom_tree_negative_volume (const t8_cmesh_t cmesh) const
+{
+  return active_tree_vertices[3] < active_tree_vertices[0] || active_tree_vertices[4] < active_tree_vertices[1]
+         || active_tree_vertices[5] < active_tree_vertices[2];
```

Comment:
I am still not sure that this is right. If exactly two axis are switched, I think the volume should be positive. Also, we should probably have a test for that

Reply:
In general you are right, but we require this condition for our axis-aligned geometry anyway, as it simplifies other computations, too. 
You're right about the test, but currently testing would be difficult, as we would need to commit a cmesh with a negative volume. @sandro-elsweijer currently works on a branch that circumvents it and as soon as it is merged I will implement a geometry-wise test for this function

Reply:
With this condition I mean, that the x-coordinate of the first point is smaller than the x-coordinate of the second (same for y and z)

Reply:
If you already require the harder condition that all coordinates are sorted, this function could just return false.

Reply:
Which other computations are simplified by disallowing switching two of the 3 axes? My intuition is that the formulas should also work for negative lengths.

Reply:
For example the point-inside check

Reply:
Ah, that makes sense

Reply:
I like that!